src/game/level.cpp

```diff
@@ -10,6 +10,185 @@ void Level::load_json(std::vector<std::shared_ptr<Entity>> *entities, const std:
   simdjson::padded_string json = simdjson::padded_string::load(real_path);
 
   this->json_data = this->json_parser.iterate(json);
+
+  simdjson::ondemand::object json_data_obj(this->json_data.get_object());
+  u32 json_data_size = json_data_obj.count_fields();
```

Comment:
you don't need this variable, just use the function directly:
`for (u32 i = 14; i < json_data_obj.count_fields() - 14; i++) {`